python-interface/src/MiniBotScripts/sound_test.py

```diff
@@ -0,0 +1,42 @@
+import Note_Library as nl
+
+TEST_EMOTION = 3;
+
+def showHappy():
+	nl.playNote(nl.G6,0.15)
+	nl.playNote(nl.A6,0.15)
```

Comment:
make a src/prototyping folder. This script is not a framework script, but it also does not comply with the user script standardss (no run(bot):) function, so it should go there